test/conformance/helpers/broker_tracing_test_helper.go

```diff
@@ -0,0 +1,489 @@
+	// 10. Transformer pod receives the event from the Broker Filter for the "transformer" trigger.
+	// 11. Broker Filter for the "transformer" sends the transformer pod's reply to the Broker
+	//     InChannel (Ingress Channel).
+	// 12. Broker InChannel receives the event from the Broker Filter for the "transformer" trigger.
+	// 13. Broker InChannel sends the event to the Broker Ingress.
+	// 14. Broker Ingress receives the event from the Broker InChannel.
+	// 15. Broker Ingress sends the event to the Broker's TrChannel.
+	// 16. Broker TrChannel receives the event from the Broker Ingress.
+	// 17. Broker TrChannel sends the event to the Broker Filter for the "transformer" trigger.
+	//     18. Broker Filter for the "transformer" trigger receives the event from the Broker
+	//        TrChannel. This does not pass the filter, so this 'branch' ends here.
+	// 19. Broker TrChannel sends the event to the Broker Filter for the "logger" trigger.
+	// 20. Broker Filter for the "logger" trigger receives the event from the Broker TrChannel.
+	// 21. Broker Filter for the "logger" trigger sends the event to the logger pod.
+	// 22. Logger pod receives the event from the Broker Filter for the "logger" trigger.
+
+	// Useful constants we will use below.
+	ingressHost := fmt.Sprintf("%s-broker.%s.svc.%s", brokerName, client.Namespace, domain)
+	ingressChanHost := fmt.Sprintf("%s-kne-ingress-kn-channel.%s.svc.%s", brokerName, client.Namespace, domain)
+	triggerChanHost := fmt.Sprintf("%s-kne-trigger-kn-channel.%s.svc.%s", brokerName, client.Namespace, domain)
+	filterHost := fmt.Sprintf("%s-broker-filter.%s.svc.%s", brokerName, client.Namespace, domain)
+	loggerTriggerPath := fmt.Sprintf("/triggers/%s/%s/%s", client.Namespace, loggerTrigger.Name, loggerTrigger.UID)
+	transformerTriggerPath := fmt.Sprintf("/triggers/%s/%s/%s", client.Namespace, transformerTrigger.Name, transformerTrigger.UID)
+	loggerSVCHost := fmt.Sprintf("%s.%s.svc.%s", loggerPodName, client.Namespace, domain)
+	transformerSVCHost := fmt.Sprintf("%s.%s.svc.%s", eventTransformerPod.Name, client.Namespace, domain)
+
+	// This is very hard to read when written directly, so we will build piece by piece.
+
+	// Steps 17-18: 'logger' event being sent to the 'transformer' Trigger.
+	loggerEventSentFromTrChannelToTransformer := tracinghelper.TestSpanTree{
+		Note: "17. Broker TrChannel sends the event to the Broker Filter for the 'transformer' trigger.",
+		Kind: model.Client,
+		Tags: map[string]string{
+			"http.method":      "POST",
+			"http.status_code": "202",
+			"http.url":         fmt.Sprintf("http://%s%s", filterHost, transformerTriggerPath),
+		},
+		Children: []tracinghelper.TestSpanTree{
+			{
+				Note: "18. Broker Filter for the 'transformer' trigger receives the event from the Broker TrChannel. This does not pass the filter, so this 'branch' ends here.",
+				Kind: model.Server,
+				Tags: map[string]string{
+					"http.method":      "POST",
+					"http.status_code": "202",
```

Comment:
`http.StatusAccepted`?
Also for all the appearances below.

Reply:
It would have to be something like `string(http.StatusAccepted)`. Personally, I feel that `"202"` is fine, but if you want the change, I will do so.

Reply:
Ah I see, then I'm also okay with both "POST" and "202"

Reply:
I'll switch to `http.MethodPost`, the problem with http.StatusAccepted is that it is an int and we would need to convert it to a string. In addition, I think people are generally less familiar with which HTTP status code is Accepted (as compared to OK or Not Found).